util/text.py

```diff
@@ -121,19 +121,22 @@ def config_file():
         return ''
 
 
-def text_to_char_array(series, alphabet):
+def text_to_char_array(transcript, alphabet, context=None):
     r"""
-    Given a Pandas Series containing transcript string, map characters to
+    Given a transcript string, map characters to
     integers and return a numpy array representing the processed string.
+    Use a string in `context` for adding text to raised exceptions.
     """
+    context = '' if context is None else 'While processing: {}\n'.format(context)
```

Comment:
Please inline the 'While processing' part back into the error messages, as it makes it easier to understand what's actually being printed. And set the default value of the `context` parameter to `''` instead of `None`.